docs/pages/guides/style.md

```diff
@@ -239,15 +240,8 @@ extend-select = [
 ]
 ignore = [
   "PLR",    # Design related pylint codes
-  "E501",   # Line too long
-  "PT004",  # Use underscore for non-returning fixture (use usefixture instead)
```

Comment:
Kind of specific, and it's not the only un-ideal PT code. Let's let users add them as they need them.